configs/bundle_config.yaml

```diff
@@ -36,11 +36,15 @@ databundles:
     category: data
     destination: "data"
     urls:
-      zenodo: https://sandbox.zenodo.org/records/3853/files/bundle_tutorial_NGBJ.zip?download=1
-      gdrive: https://drive.google.com/file/d/1Vb1ISjhy7iwTTZYeezGd6S4nLt-EDGme/view?usp=drive_link
+      zenodo: https://sandbox.zenodo.org/records/145504/files/bundle_tutorial_NGBJ_with_gadmlike.zip?download=1
+      gdrive: https://drive.google.com/file/d/12K03Epx3O9o-IQLh9afzCQyT-nMKWM3P/view?usp=drive_link
     output:
     - data/gebco/GEBCO_2021_TID.nc
     - data/copernicus/PROBAV_LC100_global_v3.0.1_2019-nrt_Discrete-Classification-map_EPSG-4326.tif
+    - data/gadm/gadm41_NGA/gadm41_NGA.gpkg
+    - data/gadm/gadm41_BEN/gadm41_BEN.gpkg
+    - data/gadm/gadm36_NGA/gadm36_NGA.gpkg
+    - data/gadm/gadm36_BEN/gadm36_BEN.gpkg
```

Comment:
As far as I remember, `gadm41` is needed for the power-only model, while `gadm36` is being used by the cross-sectoral part, right? It would be nice to have a comment on that

Reply:
Not sure it is so relevant to add these details; we have no comments on the other pieces, but I can add the lines